app/webfront/src/redux/user/slice.js

```diff
@@ -8,10 +9,19 @@ const userSlice = createSlice({
         profile: null,
         recipes: [],
         followers: [],
+        folowees: [],
         isLoading: false,
         error: null,
     },
-    extraReducers: () => {},
+    extraReducers: (builder) => {
+        builder
+            .addCase(getFollowers.fulfilled, (state, action) => {
+                state.followers = action.payload;
+            })
+            .addCase(getFollowees.fulfilled, (state, action) => {
+                state.folowees = action.payload;
+            })
+    },
 });
```

Comment:
add matchers for pending and error states, correspondent selectors and handling these states inside of pages